cmd/kpod/save.go

```diff
@@ -91,6 +91,7 @@ func saveCmd(c *cli.Context) error {
 	for _, image := range args {
 		dest := dst + ":" + image
 		if err := runtime.PushImage(image, dest, saveOpts); err != nil {
+			os.Remove(output)
```

Comment:
This can return an error, should check that

Reply:
fixed